libmozevent/storage.py

```diff
@@ -36,7 +36,7 @@ def _redis_key(self, key):
             async with AsyncRedis() as redis:
                 await redis.expire(self.name, self.expiration)
                 await redis.set(
-                    self._redis_key(key), pickle.dumps(value), expire=self.expiration
+                    self._redis_key(key), pickle.dumps(value,protocol=pickle.HIGHEST_PROTOCOL), expire=self.expiration
```

Comment:
Linting exception has be occurred.
You should correct it below 79 characters. The code of `bus.py` is the same.

Reply:
Yeap, you should install `pre-commit` and run it with  `pre-commit run -a`, it will automatically lint all the files.

Reply:
Thanks. Made the change :+1: